app/bundles/PointBundle/Model/PointModel.php

```diff
@@ -231,7 +231,9 @@ public function triggerAction($type, $eventDetails = null, $typeId = null, Lead
         foreach ($availablePoints as $action) {
             //if it's already been done, then skip it
             if (isset($completedActions[$action->getId()])) {
-                continue;
+                $properties = unserialize($completedActions[$action->getId()]['properties']);
+                if(empty($properties['page_hits']))
```

Comment:
This won't get hit because of https://github.com/mautic/mautic/pull/2296/files#diff-3f90b8c5ab009c81ced8b1d8cf545a67R104 (`===` should prevent it from being applicable since each hit will equal a ++1) but if we did allow multiple point action entries, we'd get this SQL exception `Internal Server Error - An exception occurred while executing 'INSERT INTO mautic_point_lead_action_log (date_fired, point_id, lead_id, ip_id) VALUES (?, ?, ?, ?)' with params ["2016-08-22 22:45:13", 2, 3093, 2]: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '2-3093' for key 'PRIMARY' ` because that table doesn't have an auto incremented ID (uses lead and point as the primary keys)